ui/raidboss/data/07-dt/alliance/cloud_of_darkness_chaotic.ts

```diff
@@ -0,0 +1,414 @@
+
+        delete data.bladeOfDarknessFollowup;
+
+        return output.combo!({
+          mech: mechOutput,
+          followup: output[followup]!(),
+        });
+      },
+      outputStrings: {
+        combo: {
+          en: '${mech} => ${followup}',
+        },
+        left: {
+          en: 'Left, under hand',
+        },
+        right: {
+          en: 'Right, under hand',
+        },
+        aero: {
+          en: 'Knockback',
+        },
+        death: Outputs.outThenIn,
+        out: Outputs.out,
+      },
+    },
+    {
+      id: 'Cloud Chaotic Deluge of Darkness',
+      type: 'StartsUsing',
+      netRegex: { id: ['9E3D', '9E01'], source: 'Cloud of Darkness', capture: false },
+      response: Responses.bleedAoe(),
+    },
+    {
+      id: 'Cloud Chaotic Grim Embrace Collector',
+      type: 'StartsUsing',
+      netRegex: { id: ['9E39', '9E3A'], source: 'Cloud of Darkness', capture: true },
+      run: (data, matches) => {
+        data.deadlyEmbraceDodgeDirectionCollected = matches.id === '9E39' ? 'back' : 'front';
+      },
+    },
+    {
+      id: 'Cloud Chaotic Deadly Embrace',
+      type: 'GainsEffect',
+      netRegex: { effectId: '1055', capture: true },
+      condition: Conditions.targetIsYou(),
+      preRun: (data) =>
+        data.deadlyEmbraceDodgeDirection = data.deadlyEmbraceDodgeDirectionCollected,
+      delaySeconds: (_data, matches) => parseFloat(matches.duration) - 7,
+      durationSeconds: 7,
+      infoText: (data, _matches, output) => output[data.deadlyEmbraceDodgeDirection]!(),
+      outputStrings: {
+        back: {
+          en: 'Bait hand, dodge backwards',
+        },
+        front: {
+          en: 'Bait hand, dodge forwards',
+        },
+        unknown: Outputs.unknown,
+      },
+    },
+    {
+      id: 'Cloud Chaotic Rapid-sequence Particle Beam',
+      type: 'StartsUsing',
+      netRegex: { id: '9E40', source: 'Cloud of Darkness', capture: false },
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Wild Charge (alliances)',
+        },
+      },
+    },
+    {
+      id: 'Cloud Chaotic Unholy Darkness',
+      type: 'StartsUsing',
+      netRegex: { id: 'A12D', source: 'Cloud of Darkness', capture: false },
+      infoText: (_data, _matches, output) => output.healerGroups!(),
+      outputStrings: {
+        healerGroups: Outputs.healerGroups,
+      },
+    },
+    {
+      id: 'Cloud Chaotic Flare Marker',
+      type: 'HeadMarker',
+      netRegex: { id: headMarkerData.flareMarker, capture: true },
+      condition: Conditions.targetIsYou(),
+      infoText: (_data, _matches, output) => output.flare!(),
+      outputStrings: {
+        flare: {
+          en: 'Flare on you',
+        },
+      },
+    },
+    {
+      id: 'Cloud Chaotic Break IV',
+      type: 'StartsUsing',
+      // sometimes listed as coming from Death's Hand due to actor re-use,
+      // source omitted to prevent trigger issues.
```

Comment:
This comment is misleading and I think could benefit from some additional info. There are three sets of two actors casting `Break IV`, a "primary" and an invisible clone per set:

- The boss (`9E4F`) and her clone (`9E50`)
- An add somewhere on the west (`9E51`) and its clone (`9E52`)
- An add somewhere on the east (`9E51`) and its clone (`9E52`)

`9E4F` and `9E51` casts should always have the correct actor name. If an advanced trigger were to be written to determine the positions of the two adds to give more precise instructions on where to look, those might run into problems because the invisible clones can have stale names from actor re-use (and stale position data in the `20` lines), but the base trigger here shouldn't have any issues if it's swapped to the `9E4F` cast.

Reply:
I have several examples of `9E50` (the id used here) coming from `Death's Hand` (small sample):
```
20|2024-12-26T22:10:53.1730000-06:00|4000D409|Death's Hand|9E50|Break IV|4000D409|Death's Hand|5.700|119.25|68.74|0.00|2.36|08a936ce2bcc574e
20|2024-12-26T22:22:07.4060000-06:00|4000DC5B|Death's Hand|9E50|Break IV|4000DC5B|Death's Hand|5.700|114.82|69.35|0.00|2.12|841c6c732acf42bb
20|2024-12-26T21:18:36.1510000-06:00|40002265|Death's Hand|9E50|Break IV|40002265|Death's Hand|5.700|88.66|90.74|0.00|-0.74|5d662d5756972a15
```
I also have examples of the other id using inconsistent source names:
```
20|2024-12-26T21:14:00.4190000-06:00|400020BF|Sinister Eye|9E52|Break IV|400020BF|Sinister Eye|3.700|100.00|100.00|0.00|-3.14|242872b62eebbff9
20|2024-12-26T22:05:06.0180000-06:00|4000CA89|Cloud of Darkness|9E52|Break IV|4000CA89|Cloud of Darkness|3.700|100.00|56.00|0.00|0.00|5fffc62cd2671ff6
20|2024-12-26T22:05:06.0180000-06:00|4000CA8A|Death's Hand|9E52|Break IV|4000CA8A|Death's Hand|3.700|107.55|64.81|0.00|-2.86|c418ef2aa7e20195
```
`9E4F` is more stable, but it's also just a dummy cast that ends slightly sooner, the actual effect comes from `9E50` and `9E52`.